server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  attributes: {
+    id: {
+      type: 'string',
+      columnName: '_id'
+    },
+    createdAt: {
+      type: 'number',
+      autoCreatedAt: true,
+    },
+    updatedAt: {
+      type: 'number',
+      autoUpdatedAt: true,
+    },
+    fullName: {
+      type: 'string',
+      required: true
+    },
+    email: {
+      type: 'string',
+      required: true,
+      unique: true,
+      isEmail: true
+    },
+    campus: {
+      type: 'string',
+      required: true
+    },
+    lifestage: {
+      type: 'string',
+      required: true
+    },
+    isMember: {
+      type: 'ref',
```

Comment:
why is this type ref instead of boolean?

Reply:
any reason why this is not required?

Reply:
i think i was trying to make it tied to the membership table, but after thinking about it more, I'll just have it boolean and false as default
it's not required because I'm not sure if it'll be a required field up in the sign up process for frontend, so i will just make it false for now